### ISSUE TITLE
Is it possible to wait until browser custom protocol confirmation to be accepted?

### ISSUE THREAD

**Issue by kirill-konshin** [ismailhabib/custom-protocol-detection#31]
When browser is redirected to custom protocol link it shows the custom protocol handler confirmation dialog like this one in Safari:

![image](https://user-images.githubusercontent.com/1530410/29947652-ab06a7a2-8e5f-11e7-8790-3a1d759a4f8f.png)

Unfortunately, right now the lib does not capture this, e.g. fires success callback right away when the dialog is shown (assuming that protocol has been handled).

In reality if user refuses to launch the app some extra actions may be needed so we should somehow detect acceptance and fire some callbacks.

**Comment by kirill-konshin** [comment]
Bump :)

**Comment by ismailhabib** [comment]
Not really, unless Safari starts implementing something like `msLaunchUri`.

**Comment by kirill-konshin** [comment]
Shall we submit the issue for them?
